src/Simplex/Chat/Controller.hs

```diff
@@ -1068,6 +1075,16 @@ instance FromJSON ComposedMessage where
   parseJSON invalid =
     JT.prependFailure "bad ComposedMessage, " (JT.typeMismatch "Object" invalid)
 
+data ChatTagData = ChatTagData
+  { emoji :: Text,
+    text :: Text
+  }
+  deriving (Show)
+
+instance FromJSON ChatTagData where
+  parseJSON (J.Object v) = ChatTagData <$> v .: "emoji" <*> v .: "text"
+  parseJSON invalid = JT.prependFailure "bad ChatTagData, " (JT.typeMismatch "Object" invalid)
```

Comment:
why do we need custom instance and not automatic with defaultOptions?

Reply:
we may need a different parser for CLI commands, but for API could be just JSON.